src/preamble.js

```diff
@@ -930,11 +930,11 @@ function stackTrace() {
 
 // Memory management
 
-var PAGE_SIZE = 4096;
+var PAGE_SIZE = 65536; // Wasm pages are fixed to 64k.
```

Comment:
this changes not just how we allocate, but how we report our filesystem page size. 4096 was copied from linux, I believe, to minimize risk of incompatibility errors. are we sure this change will work?